app/views/users/registrations/edit.html.slim

```diff
@@ -51,6 +51,14 @@
               autocomplete: 'new-password',
               class: 'form-control'
 
+    .form-group.field-element
+      = f.label :personal_openai_api_key, class: 'form-label'
```

Comment:
The label is missing proper capitalization:

![image](https://github.com/openHPI/codeharbor/assets/24592181/ecde2953-d73f-4d49-ab2a-fb057f86d08e)